app/models/ca_search_forms.php

```diff
@@ -996,6 +996,44 @@ public function getAvailableBundles($pm_table_name_or_num=null, $pa_options=null
 			);
 		}
 
+		// Hidden Field
+		$va_additional_settings = array(
+			'hidden_field_name' => array(
+				'formatType' => FT_TEXT,
+				'displayType' => DT_FIELD,
+				'width' => 25, 'height' => 1,
+				'takesLocale' => false,
+				'required' => true,
+				'label' => _t('Field Name'),
+				'description' => _t('Name of the hidden input field. For example <em>ca_objects.type_id</em>.'),
+			),
+			'hidden_field_value' => array(
+				'formatType' => FT_TEXT,
+				'displayType' => DT_FIELD,
+				'width' => 25, 'height' => 1,
+				'takesLocale' => false,
+				'requireValue' => true,
+				'label' => _t('Value'),
+				'description' => _t('Value of the hidden input field.')
+			)
+		);
+		$t_placement->setSettingDefinitionsForPlacement($va_additional_settings);
+		$vs_bundle = 'hidden_field';
+		$vs_display = "<div id='searchFormEditor_$vs_bundle'><span class='bundleDisplayEditorPlacementListItemTitle'>"._t('Custom').'</span> '.($vs_label = _t('Hidden Field'))."</div>";
+		$va_available_bundles[strip_tags($vs_display)][$vs_bundle] = array(
+			'bundle' => $vs_bundle,
+			'label' => $vs_label,
+			'display' => $vs_display,
+			'description' => $vs_description = _t('Provides a hidden search field on the search form.'),
+			'settingsForm' => $t_placement->getHTMLSettingForm(array('id' => $vs_bundle.'_0')),
```

Comment:
Again consistent with other bundles. Ones that are tied to a placement have the placement id appended there.